tests/wpunit/TEC/Tests/Events/Admin/Notice/RESTAPITest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * REST API Notice test.
+ */
+
+declare( strict_types=1 );
+
+namespace TEC\Tests\Events\Admin\Notice;
+
+use Closure;
+use Codeception\TestCase\WPTestCase;
+use TEC\Events\Admin\Notice\Rest_Api;
+use WP_Error;
+
+/**
+ * Class RESTAPITest
+ *
+ * @since TBD
+ */
+class RESTAPITest extends WPTestCase {
+
+	function test_response_blocked_from_wp_error() {
+		$request_return_timeout = function() {
+			return new WP_Error(
+				'http_request_failed',
+				'cURL error 28: Operation timed out after 10000 milliseconds with 0 bytes received'
+			);
+		};
+
+		$request_return_ssl_error = function() {
+			return new WP_Error(
+				'http_request_failed',
+				'cURL error 60: SSL certificate problem: unable to get local issuer certificate'
+			);
+		};
+
+		$rest_api = new Rest_Api();
+
+		// Set up a wrapper to test the private method.
+		$is_response_blocking = Closure::bind(
+			function( $response ) {
+				return $this->is_wp_error_response_blocking( $response );
+			},
+			$rest_api,
+			$rest_api
+		);
```

Comment:
Please replace with using the `With_Uopz` trait and its method 

`$this->set_fn_return( 'wp_safe_remote_get', $request_return_timeout, true );`

Then you could actually execute the actual method by just faking the return of `wp_safe_remote_get`